requirements-dev.txt

```diff
@@ -0,0 +1,4 @@
+black>=23.1.0
```

Comment:
Pre-commit installs a separate instance of black that it uses when it runs its automatic checks. That instance of black is not present in the PATH during routine coding. I'll remove from this location, and we can leave it up the user to manually install it if they want to run black manually instead of using pre-commit hooks.